package-lock.json

```diff
@@ -773,7 +773,8 @@
           "version": "3.0.0",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-3.0.0.tgz",
           "integrity": "sha1-7QMXwyIGT3lGbAKWa922Bas32Zg=",
-          "dev": true
+          "dev": true,
```

Comment:
Since there are no changes in dependencies, please, revert changes for this file

Reply:
@aefox